lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -116,12 +120,10 @@ def test_statsmodels_mean(self, warning_list=None):
             historic_forecasts.append(temp_cube)
         historic_forecasts.concatenate_cube()
 
-        distribution = "gaussian"
-        desired_units = "degreesC"
         predictor_of_mean_flag = "mean"
 
-        if not statsmodels_found:
-            Plugin(distribution, desired_units,
+        if not STATSMODELS_FOUND:
+            Plugin(self.distribution, self.desired_units,
```

Comment:
Which warning is this catching? And should this be a skip thing as well (and should some of the later tests be skipped too)?